tools/fuchsia/flutter/internal/flutter_dart_component.gni

```diff
@@ -220,6 +239,9 @@ template("flutter_dart_component") {
 
     # copy the snapshot as a resource
     _snapshot_resource_target_name = "${target_name}_snapshot_resource"
+
+    # TODO(richkadel): It looks like this is the _ONLY_ place resource() is used
```

Comment:
Yes, I left it in for now (but with the TODO) so the structure of files derived from Fuchsia is as similar to the original as possible. (As mentioned in my earlier comment, this helps see what did and did not change. I figure it can be optimized in a subsequent PR once we're happy that everything else is working as expected.